### ISSUE TITLE
Hexadecimal size calculation problems

### ISSUE THREAD

**Issue by anthonybastidas49** [moov-io/iso8583#120]
Hello, I have some problems in the coding of the prefixes, since the size of the variable fields first converts them from int to string and from there to hexadecimal for which reason. If I have a size of 8 at the moment the library puts it in hexadecimal encoding as "3338". which should be 8.
Additionally it is counting the size two by two and not one by one

**Comment by wadearnold** [comment]
Thanks, @anthonybastidas49 for pointing this out. Can you make a test case that has the expected result that fails? If you have coded a work around we are happy to merge a pull request. We will get to this issue as soon as we can.

**Comment by alovak** [comment]
Hi @anthonybastidas49!
May I ask you to share some more information with us so we can fix the issue?

**Comment by anthonybastidas49** [comment]
Yes, the calculation of the sizes for visa frames. in fields of variable length it is calculating it wrongly.

**Comment by alovak** [comment]
@anthonybastidas49 without any examples that demo the problem it's really hard for us to fix the issue :(

Can you give us an example where calculation does not work?

**Comment by alovak** [comment]
@anthonybastidas49 if this issue is still relevant, let discuss it in Moov's slack channel [#iso8583](https://moov-io.slack.com/archives/C014UT7C3ST).

We can't fix the issue without more information. You can use different encodings for prefixes. If you don't need them to be converted to hex, use ASCII prefixer like this:

```
2: field.NewString(&field.Spec{
	Length:      19,
	Description: "Primary Account Number",
	Enc:         encoding.ASCII,
	Pref:        prefix.ASCII.LL,
}),
```

**Comment by alovak** [comment]
I hope that it was solved by #193

if no, we can re-open the issue
